# Parameter Configuration File for MPC Trajectory Generator

# ROS Topic Names
pose_topic: "pf/pose/odom"
scan_topic: "scan"
drive_topic: "nav"

# Visualization
visualization_enabled: true
obstacle_avoidance: true

# Input CSV File Path
csv_path: "/home/yash/yasht_ws/src/mpc_av/sensor_data/gtpose_fast.csv"
csv_trajectories_path: "/home/yash/yasht_ws/src/mpc_av/trajectories.csv"

# Tracking Parameter
lookahead_distance: 1.5

# WayPoint Input Method
# Options- 'user' - Multiple Inputs using interactive markers
#          'raceline' - PreComputed Raceline WayPoints
waypoint_input: "raceline"

# Local Map Parameter
local_map_length: 1.5
inflation_radius: 4 #cells